src/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -128,7 +128,7 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.DeleteVolumeMountPoint.cs">
       <Link>Common\Interop\Windows\Interop.DeleteVolumeMountPoint.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\kernel32\Interop.GetLogicalDrives.cs">
+    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Kernel32\Interop.GetLogicalDrives.cs">
```

Comment:
Should we instead be consistent and rename this directories to be all lowercase? It is odd to have someones to be lowercase and other uppercase.